etc/behat/services/contexts.xml

```diff
@@ -22,6 +22,7 @@
         <parameter key="sylius.behat.context.setup.product.class">Sylius\Behat\Context\Setup\ProductContext</parameter>
         <parameter key="sylius.behat.context.setup.promotion.class">Sylius\Behat\Context\Setup\PromotionContext</parameter>
         <parameter key="sylius.behat.context.setup.shipping.class">Sylius\Behat\Context\Setup\ShippingContext</parameter>
+        <parameter key="sylius.behat.context.setup.review.class">Sylius\Behat\Context\Setup\ReviewContext</parameter>
```

Comment:
I think it should be named ProductReviewContext and service id adjusted as well.